src/os/unix/unix.cpp

```diff
@@ -288,7 +292,20 @@ bool GetClipboardContents(char *buffer, const char *last)
 #endif
 
 
-#ifndef __APPLE__
+#if defined(__EMSCRIPTEN__)
+EM_JS(void, em_open_in_browser, (const char *url, int url_len), {
+	const url_string = UTF8ToString(url, url_len);
+	function openWindow() {
+		document.removeEventListener("mouseup", openWindow);
+		window.open(url_string, '_blank');
+	}
+	document.addEventListener("mouseup", openWindow);
```

Comment:
Why do this only on mouse-up? Is there a reason for it?

Reply:
This is done to make sure that `window.open` isn't called before OpenTTD registers the mouse going up. Without this the on-screen button remains stuck down when you navigate back to OpenTTD.

Reply:
Please always mentions these kind of things in comments. The fact that I ask, means it is not obvious, but your explanation is also not something I would have guessed :D I always think: would the future-me in 2 years still remember why I did it this way? No? Explain it with a comment :D Tnx :)

Reply:
Noted in `pre.js`.